staging/src/k8s.io/apiserver/pkg/endpoints/request/webhook_duration.go

```diff
@@ -36,8 +36,19 @@ func maxDuration(d1 time.Duration, d2 time.Duration) time.Duration {
 }
 
 // DurationTracker is a simple interface for tracking functions duration
+// DurationTracker can be safely used concurrently.
+// Whether the specified function (that is invoked by the DurationTracker)
```

Comment:
meaning it must be safe to access concurrently or callers cannot know?  seems more useful if we declare they must be safe to access concurrently.

Reply:
it is safe to access it from multiple go routines.
i wanted to state that whether `f` is safe to execute concurrently rests with the caller. comments updated